Grundgeruest/models.py

```diff
@@ -177,7 +177,7 @@ class ScholariumProfile(UserenaBaseProfile):
         max_length=30,
         null=True, blank=True)
     plz = models.CharField(
-        max_length=5,
+        max_length=10,
         validators=[RegexValidator('^[0-9]+$')],
```

Comment:
Hat nichts mit der geänderten Länge zu tun, ist mir aber hier aufgefallen und könnte in diesem PR noch ergänzt werden:

Es gibt auch alphanumerische PLZs, z.B. in GB. Insofern sollte der Validator auch Buchstaben und Bindestriche sowie Leerzeichen enthalten. Ich würde ihn dann gleich rausnehmen (auch wenn die große Mehrheit nur numerische PLZ hat. Oder übersehe ich etwas?

Reply:
Guter Punkt.